src/Grinder/Commands.fs

```diff
@@ -298,7 +308,11 @@ module Processing =
                 let! requestsResult = 
                     [for chat in botSettings.ChatsToMonitor.Set do
                         for user in usernames do
-                            yield ApiExt.restrictUser context.UpdateContext chat user time]
+                            match (botSettings, user) with
+                            | UserCanBeBanned ->
+                                yield ApiExt.restrictUser context.UpdateContext chat user time
+                            | UserCannotBeBanned -> 
+                                sprintf "Cannot ban admin %s" user |> ignore]
```

Comment:
Wait, what? `sprintf |> ignore` won't do anything. You should definitely report something here, and `ignore` is just `ignore`.